### PR TITLE
Rename tasks to have test id and description

### PR DESCRIPTION
**Pull request by elfiesmelfie**
The tasks were previously renamed to just be the test id, due to limitations in the custom_logger callbaack plugin. This was addressed, and the test_report_result reports in the correct format now, so the names can be reverted.
This makes reports more grokable, since the summary_result.log contains a list of the passed/failed tests with the task names.

Depends-On: https://github.com/infrawatch/feature-verification-tests/pull/177

### CODE REVIEW

File: roles/test_collectd/tasks/main.yml
Comment (ayefimov-1) [inline roles/test_collectd/tasks/main.yml:1]: Do the name strings have to be double qouted?  This file and qdr_test/tasks/main.yml file names are not double qouted!

Comment (elfiesmelfie) [reply roles/test_collectd/tasks/main.yml:1]: Unless they have some special characters, they don't have to be quoted.
I kept in quotes consistent with the file that the task is in

**Comment by elfiesmelfie** [comment]
recheck

**Comment by elfiesmelfie** [comment]
recheck

**Review by vkmc — APPROVED** [review]
So nice to be able to have the description with the ID... definitely easier to debug in the long run. LGTM, thanks!

**Pull request merged**